spec/acceptance/sequence_spec.rb

```diff
@@ -1,72 +1,166 @@
 describe "sequences" do
   include FactoryBot::Syntax::Methods
+  require "ostruct"
 
-  it "generates several values in the correct format" do
-    FactoryBot.define do
-      sequence :email do |n|
-        "somebody#{n}@example.com"
+  # = On Success
+  # ======================================================================
+  #
+  describe "on success" do
```

Comment:
I believe this (and others) could be better read and more correct semantically as `context`